OpenRA.Mods.Common/Traits/Render/WithProductionOverlay.cs

```diff
@@ -49,7 +49,7 @@ public class WithProductionOverlay : INotifyDamageStateChanged, INotifyCreated,
 
 		bool IsProducing
 		{
-			get { return queues != null && queues.Any(q => q.Enabled && q.CurrentItem() != null && !q.CurrentPaused); }
+			get { return queues != null && queues.Any(q => q.Enabled && q.AllQueued().Any(i => !i.Paused)); }
```

Comment:
This is another case that needs to filter out units that haven't started yet.

It breaks the welding animations in d2k if you start producing a vehicle (animation shows), pause it (animation correctly pauses), then queue a different unit behind it (animation incorrectly restarts).